### DISCUSSION TITLE
Portuguese language

### DISCUSSION THREAD

**Discussion by 388759** [Taewan-P/gpt_mobile#69] [Ideas]
Hello, I recently discovered the app and I loved it. Is there a possibility of adding the Portuguese language?

**Comment by Taewan-P** [comment]
Hello. Thanks for using our app!
Starting today, [Hosted Weblate](https://hosted.weblate.org/engage/gptmobile/) is available for this project. I hope anyone who speaks Portuguese can contribute to the project!
